src/view/components/grid/grid.js

```diff
@@ -138,8 +143,15 @@ export class Grid extends RootComponent {
 		return this.model.visibility();
 	}
 
-	get isActive() {
-		return this.table.view.isFocused();
+	invalidateIsActive() {
```

Comment:
Maybe just `invalidateActive`?

Reply:
maybe even better name?

Reply:
What is better? onFocusChanged or invalidateActive?